app/services/LocationService.js

```diff
@@ -258,6 +258,17 @@ export default class LocationServices {
     });
   }
 
+  // To get the system dialog click event
+  static async getSystemLocationPermission() {
```

Comment:
is this being used anywhere?

Reply:
No. Only one time in the settings page.

Reply:
I still don't quite understand why this is needed. There is no callback that is passed to the authorization handler, so nothing will ever be done